src/webapp/azext_webapp/tunnel.py

```diff
@@ -84,7 +84,7 @@ def is_port_set_to_default(self):
 
         http = urllib3.PoolManager(cert_reqs='CERT_REQUIRED', ca_certs=certifi.where())
         headers = urllib3.util.make_headers(basic_auth='{0}:{1}'.format(self.remote_user_name, self.remote_password))
-        url = 'https://{}{}'.format(self.remote_addr, '.scm.azurewebsites.net/AppServiceTunnel/Tunnel.ashx?GetStatus')
+        url = '{}{}'.format(self.remote_addr, '/AppServiceTunnel/Tunnel.ashx?GetStatus')
```

Comment:
We enforce https for scm site in prod

Reply:
this just uses whatever is returned by get_scm_url.